src/main/java/edu/oakland/production/display02/DisplayCommManagerImplementation.java

```diff
@@ -0,0 +1,64 @@
+package edu.oakland.production.display02;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.display02.DisplayCommManager;
+import edu.oakland.production.middleware02.MiddlewareCommInterface;
+import java.lang.IllegalArgumentException;
+
+/**
+* This class receives the rfid, evaluates it,
+* then passes it along to Middleware
+*
+*/
+public class DisplayCommManagerImplementation implements DisplayCommManager {
+
+    private MiddlewareCommInterface middleware;
+    public int rfid;

```

Comment:
According to the UML Class Diagram, you shouldn't need this RFID number as it is just a pass through class.
```suggestion

```